### ISSUE TITLE
Forge dynamic bucket Z-fighting when rendered on drawers

### ISSUE THREAD

**Issue by KnightMiner** [jaquadro/StorageDrawers#382]
![2016-12-06_21 47 37](https://cloud.githubusercontent.com/assets/12155288/20954070/a207457e-bbfd-11e6-90aa-9aaed6884c06.png)

As shown in the picture, the Forge dynamic bucket (and the bucket from my mod which uses the same model) has Z-fighting when rendered on a drawer.

Using Storage Drawers 1.10.2-3.5.7 and reproducible with only Storage Drawers installed.

**Comment by jaquadro** [comment]
I wonder if it's a case where the forge icon will render reliably in a perfectly head-on display (the GUI) but not at off-angles as you see in the world.  I'm not sure I can do anything to fix that.

**Comment by codewarrior0** [comment]
Maybe something goofy like rendering the item icon to an offscreen texture, and then drawing that onto the drawer.

**Comment by jaquadro** [comment]
Are most or all mod buckets derived from dynamic (the items all appear to be attributed to MinecraftForge)?  I just tested a bunch and I didn't see any Z-fighting.

Historically I've seen Z-fighting become an issue in the presence of Optifine.

**Comment by KnightMiner** [comment]
Yeah, just about all fluids these days use the forge bucket, though note that in a fresh Forge install it is disabled by default, mods like Tinkers enable the Forge switch so it remodels vanilla fluids as well (there is a boolean in the Forge config file to test it without Tinkers). From what I know from a similar case, it comes from quads not being perfectly aligned after transformations, but it is odd that it happens on buckets and not vanilla multilayer items then.

----

I never use Optifine, just about every other update breaks something simple in Tinkers Construct or Immersive Engineering, leaving me to just avoid it.

**Comment by tofof** [comment]
Yup, having this issue too. I'm not using Optifine.
